### PR TITLE
Adding function to retrieve evaluation measures

### PR DESCRIPTION
**Pull request by Neeratyoy**
#### Reference Issue

Addresses #687.

#### What does this PR implement/fix? Explain your changes.
Adds a new function to get a list of evaluation measures.

#### How should this PR be tested?
`openml.evaluations.list_evaluation_measures()`

#### Any other comments?
Have placed it in `openml/evaluations/functions.py` file. That seemed like the appropriate location for this function. Kindly advise if not.

### CODE REVIEW

File: doc/progress.rst [hunk 1]
```diff
@@ -21,6 +21,7 @@ Changelog
 * ADD #659: Lazy loading of task splits.
 * ADD #516: `run_flow_on_task` flow uploading is now optional.
 * ADD #680: Adds `openml.config.start_using_configuration_for_example` (and resp. stop) to easily connect to the test server.
+* ADD #687: Adds a function to retrieve the list of evaluation measures available.
```
Comment (PGijsbers) [inline doc/progress.rst]: Please add new additions under a new header (for now we assunme 0.10.0). The 0.9.0 release is already out.

File: openml/evaluations/functions.py [hunk 2]
```diff
@@ -200,3 +200,29 @@ def __list_evaluations(api_call, output_format='object'):
         evals = pd.DataFrame.from_dict(evals, orient='index')
 
     return evals
+
+
+def list_evaluation_measures() -> list:
```
Comment (PGijsbers) [inline openml/evaluations/functions.py]: Specify return type as `List[str]`.

**Comment by PGijsbers** [comment]
As indicated earlier (sorry, I think I only put it as a comment), please add new additions under a new header (just call the header 'newest release' for now). The 0.9.0 release is already out, so this change is not contained in it.

**Comment by Neeratyoy** [comment]
Sorry about that. I guess I overlooked that. Have put 0.10.0 for now as you mentioned in the initial comment.

**Comment by PGijsbers** [comment]
Thank you for your contributions btw 🎉

**Comment by Neeratyoy** [comment]
my pleasure, and learning a lot!
thanks :blush:

**Pull request merged**